src/tokenManagement/tokenManagementTransactionsFactory.ts

```diff
@@ -440,44 +362,44 @@ export class TokenManagementTransactionsFactory {
 
         return new TransactionBuilder({
             config: this.config,
-            sender: options.sender,
-            receiver: options.sender,
+            sender: sender,
+            receiver: sender,
             dataParts: dataParts,
             gasLimit: this.config.gasLimitEsdtNftCreate + storageGasLimit,
             addDataMovementGas: true,
         }).build();
     }
 
-    createTransactionForPausing(options: { sender: IAddress; tokenIdentifier: string }): Transaction {
+    createTransactionForPausing(sender: IAddress, options: resources.ManagementInput): Transaction {
```

Comment:
`resources.ManagementInput` also has a mandatory `user: IAddress` field that this method doesn't need.

Reply:
good catch :+1: